.github/workflows/build.yaml

```diff
@@ -22,7 +27,7 @@ jobs:
       # When you want to use a string variable as a workflow YAML boolean, it's
       # important to ensure it's the empty string when false. If you omit || '',
       # its value when false is "false", which is interpreted as true.
-      RELEASE_RUN: ${{ github.ref_type == 'tag' && startsWith(github.ref_name, 'Second_Life') && 'Y' || '' }}
+      RELEASE_RUN: ${{ (github.event.inputs.release_run || github.ref_type == 'tag' && startsWith(github.ref_name, 'Second_Life')) && 'Y' || '' }}
```

Comment:
I think (but am not sure, in workflow YAML expression land) that `&&` binds more closely than `||`. That means your parens are correct as written -- but might be clearer to the casual reader if you additionally parenthesized the original `(... == 'tag' && startsWith(...))` expression.